opentracing/src/main/java/io/opentracing/Span.java

```diff
@@ -52,6 +57,14 @@
    * Returns null if no entry found, or baggage is not supported in the current implementation.
    */
   String getBaggageItem(String key);
+  
+  /**
+   * Useful for acquiring the set of baggage keys so that clients of the Span have the ability
+   * to access the corresponding baggage value.
+   * 
+   * @return {@link Set} of baggage keys
+   */
+  Set<String> getBaggageKeys();
```

Comment:
Do you have an example of when and why this would be used?
I'm unsure of a situation where you'd need to explore what baggage existed.